trips/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Deck.gl - Carto Trips example</title>
+  </head>
+  <body>
+    <div id="map"></div>
+    <canvas id="deck-canvas"></canvas>
+    <div id="story-card">
+      <p class="overline">✨👀 You're viewing</p>
+      <h2>Deck.gl - Carto Trips example</h2>
+      <p>Request data directly using the <b>CARTO SQL API</b> to integrate it with other deck.gl layers</p>
```

Comment:
Adapt the following description

Reply:
I changed it, but I'm unsure if it will be the final description.
Any suggestion?